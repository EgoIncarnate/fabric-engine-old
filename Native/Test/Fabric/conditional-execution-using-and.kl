function doSomething()
{
  report "Did something";
}

function doSomethingElse()
{
  report "Did something else";
}


function entry()
{
  true && doSomething();
  false && doSomethingElse();
}
