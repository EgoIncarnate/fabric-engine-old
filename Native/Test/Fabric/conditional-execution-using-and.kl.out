conditional-execution-using-and.kl:3: SyntaxError: Unexpected string
  report "Did something";
         ^^^^^^^^^^^^^^^
